CHANGELOG.md

```diff
@@ -10,7 +10,13 @@ All notable changes to this project will be documented in this file.
   - `podOverrides`
   - `affinity`
 
+### Fixed
+
+- BREAKING: The fields `connection` and `host` on `S3Connection` as well as `bucketName` on `S3Bucket`are now mandatory ([#121]).
```

Comment:
So far we only mentioned the S3 changes for operators actually using S3, which hello-world does not. I would be in favor of not having a changelog entry, but please feel free to mention the logging thing anyway